### DISCUSSION TITLE
Tidal x Roon rich prescence

### DISCUSSION THREAD

**Discussion by abc6d19** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#93] [Ideas]
Would there ever be a possibility of getting Roon to work with this since it uses tidal to play music?

**Comment by purpl3F0x** [comment]
It's possible and not so hard to implement tbh since roon has an API, although i don't use roon anymore so there is that - don't feel like paying for just making that.
If there is enought intrestrest I guess....

**Reply by abc6d19** [reply]
I understand, Someone made a program to try and create discord rich presence for roon, this is the link to it, not sure if it looks good or works properly, would love to get your thoughts on it https://github.com/jtpox/discord-presence-roon

**Comment by purpl3F0x** [comment]
What should be my thoughts?
If it works, it works I guess
